lib/tirexs/bulk.ex

```diff
@@ -50,7 +50,7 @@ defmodule Tirexs.Bulk do
 
       header = [_index: index, _type: type, _id: id ]
       if retry_on_conflict do
-        header = Dict.put(header, :retry_on_conflict, retry_on_conflict)
+        header = Dict.put(header, :_retry_on_conflict, retry_on_conflict)
```

Comment:
According, https://www.elastic.co/guide/en/elasticsearch/reference/2.x/docs-bulk.html#bulk-update - this field should be used for `update` action. It looks like a regression.